msteams-platform/m365-apps/extend-m365-teams-message-extension.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Extend a Teams message extension across Microsoft 365
+description: Here's how to update your search-based Teams messaging extension to run in Outlook
+ms.date: 11/15/2021
+ms.topic: tutorial
+---
+# Extend a Teams message extension across Microsoft 365 (preview)
+
+> [!NOTE]
+> Features included in preview may not be complete, and may undergo changes before becoming available in the public release. They are provided for testing and exploration purposes only. They should not be used in production applications.
```

Comment:
To maintain consistency with other pages, could we please remove "(preview)" from the title and add it to the note:

> [!NOTE]
> Extend a Teams message extension across Microsoft 365 is currently available only in [public developer preview](../../../resources/dev-preview/developer-preview-intro.md). Features included in preview may not be complete, and may undergo changes before becoming available in the public release. They are provided for testing and exploration purposes only. They should not be used in production applications.

Reply:
Updated in next commit.